base/Enums.jl

```diff
@@ -9,6 +9,9 @@ abstract Enum
 
 Base.convert{T<:Integer}(::Type{T}, x::Enum) = convert(T, box(Int32, x))
 
+Base.:(==)(x::Enum, y::Integer) = Int32(x) == y
+Base.:(==)(x::Integer, y::Enum) = x == Int32(y)
```

Comment:
Done. These lines were broken from the start, so that's the first bug uncovered by the new strict rules.